modules/hcp-cli-console.adoc

```diff
@@ -7,6 +7,10 @@
 
 You can install the {hcp} command-line interface (CLI), `hcp`, by using the {product-title} web console.
 
+.Prerequisites
+
+* You have installed {mce} 2.5 or later on an {product-title} cluster. The {mce-short} is automatically installed when you install Red{nbsp}Hat Advanced Cluster Management. You can also install {mce-short} without Red{nbsp}Hat Advanced Management as an Operator from {product-title} OperatorHub.
```

Comment:
I'm curious if this is a standard to specify "version X.Y or later" . I mean, specifically the word "later". In this sentence it might also belong to the second part of the sentence: "later on an OpenShift Container Platform cluster". I had to read the sentence a couple of times to completely understand it (maybe just my own problem as I'm not a native speaker). Is it possible to replace the word "later" with "newer" or something else?
This is just a nit. Please let me know if I should just ignore this :) Thanks

Reply:
@mgencur Good question. The typical terminology for referring to versions is "version X.Y or later". I can see how the wording here might make you stumble, though. Would it be clearer if I reworded it to something like this?
`On an OpenShift Container Platform cluster, you have installed MCE 2.5 or later.`

Reply:
Yeah, that would be clearer. IMO. Thanks